index.html

```diff
@@ -480,13 +480,31 @@ <h4 id="additional-definitions-">Additional Definitions</h4>
         compliant implementations satisfy the semantic
         guarantees that are implied by them.
       </dd>
+
+      <dt>
+        <dfn> Baseline Information Model</dfn>
+      </dt>
+      <dd>
+         Synonym for <a>HTTP Baseline Information Model</a>
+      </dd>
+      
+      <dt>
+        <dfn>Baseline Profile</dfn>
+      </dt>
+      <dd>
+          Synonym for <a>HTTP Baseline Profile</a>
+      </dd>
+
+
       <dt>
         <dfn>HTTP Baseline Profile</dfn>
       </dt>
       <dd>The subset of the Thing Description
-        defined by the present document.</dd>
+        defined by the present document.
```

Comment:
it is not a subset, more than that currently. Also, many parts that has nothing to do with TDs (how actions work for example). I would say something like "Requirements on how to implement HTTP using Things in a certain way in the Web of Things"